$OpenBSD: patch-rights,v 1.1.1.1 2000/03/27 08:32:24 camield Exp $

This port wants to change the rights on PREFIX/ and PREFIX/bin to 02755.
Usually PREFIX is /usr/local. The rights on /usr/local and /usr/local/bin
should be 0755 according to /etc/mtree/4.4BSD.dist.

Also, use /usr/local/share instead of /usr/local/etc.

--- hier.c.orig	Sat Feb 26 03:03:17 2000
+++ hier.c	Tue Feb 29 08:22:33 2000
@@ -2,13 +2,13 @@
 
 void hier()
 {
-  h(auto_home,-1,-1,02755);
-  d(auto_home,"etc",-1,-1,02755);
-  d(auto_home,"etc/dnscache",-1,-1,02755);
-  d(auto_home,"bin",-1,-1,02755);
+  h(auto_home,-1,-1,0755);
+  d(auto_home,"share",-1,-1,0755);
+  d(auto_home,"share/dnscache",-1,-1,0755);
+  d(auto_home,"bin",-1,-1,0755);
 
-  c(auto_home,"etc/dnscache","VERSION",-1,-1,0644);
-  c(auto_home,"etc/dnscache","@",-1,-1,0644);
+  c(auto_home,"share/dnscache","VERSION",-1,-1,0644);
+  c(auto_home,"share/dnscache","@",-1,-1,0644);
 
   c(auto_home,"bin","dnscache-conf",-1,-1,0755);
   c(auto_home,"bin","tinydns-conf",-1,-1,0755);
